### PR TITLE
Add version information and github url

### LINKED ISSUE #29: Adding GitHub page to settings.
Adding this GitHub page to the etc section in the settings would be convenient.

### PR DESCRIPTION
**Pull request by hskrtich**
Add version information and github url.

This should close #29 and #31

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -12,7 +12,7 @@ android {
         minSdkVersion 21
         targetSdkVersion 30
         versionCode 21
-        versionName "0.45.1"
+        versionName "0.51.2"
```
Comment (mattttvaughn) [inline app/build.gradle:15]: much appreciated 👍

File: app/src/main/java/io/github/mattpvaughn/chronicle/features/settings/SettingsViewModel.kt [hunk 2]
```diff
@@ -553,6 +553,24 @@ class SettingsViewModel(
...
+                click = object : PreferenceClick {
+                    override fun onClick() {
+                        _webLink.postEvent("https://github.com/mattttvaughn/chronicle")
+                    }
+                }),
+            PreferenceModel(
+                type = PreferenceType.CLICKABLE,
+                title = FormattableString.from(R.string.settings_version_title),
+                explanation = FormattableString.from(BuildConfig.VERSION_NAME),
+                click = object : PreferenceClick {
```
Comment (mattttvaughn) [inline app/src/main/java/io/github/mattpvaughn/chronicle/features/settings/SettingsViewModel.kt]: don't think we need this click here. but nothing to sweat, isn't hurting anything.

I'm happy to merge this in as it is as long as you weren't planning on using it for anything

**Review by mattttvaughn — APPROVED** [review]
looks good! one minor comment but happy with this if you are 

and didn't call it out but a 👍 for the gitignore stuff too

File: app/src/main/java/io/github/mattpvaughn/chronicle/features/settings/SettingsViewModel.kt (hunk 2 above)
Comment (hskrtich) [reply app/src/main/java/io/github/mattpvaughn/chronicle/features/settings/SettingsViewModel.kt]: I looked at the available options under PreferenceType and I didn't see a better option. I'm happy to change it, I wasnt planning on using it.

**Comment by hskrtich** [comment]
I almost added the whole .idea folder to gitignore since im pretty sure thats standard practice for shared projects but that a larger change then i felt comfortable with. I figured i would reach out to you about that at some point to get your opinion

**Comment by mattttvaughn** [comment]
> I almost added the whole .idea folder to gitignore since im pretty sure thats standard practice for shared projects but that a larger change then i felt comfortable with. I figured i would reach out to you about that at some point to get your opinion

Fine by me, definitely fought .idea/ the last few PRs. If you don't do it I'll change it myself sometime soon

**Comment by hskrtich** [comment]
Cool, i will submit a new PR for .idea ignore

File: app/src/main/java/io/github/mattpvaughn/chronicle/features/settings/SettingsViewModel.kt (hunk 2 above)
Comment (mattttvaughn) [reply app/src/main/java/io/github/mattpvaughn/chronicle/features/settings/SettingsViewModel.kt]: There's isn't a `NON_CLICK` or `VIEW` PreferenceType or anything, but PreferenceModel provides an empty `click` by default so you don't need to explicitly pass an empty one

Comment (hskrtich) [reply app/src/main/java/io/github/mattpvaughn/chronicle/features/settings/SettingsViewModel.kt]: Ahh okay, I have not done android development before. Let me get that update

Comment (mattttvaughn) [reply app/src/main/java/io/github/mattpvaughn/chronicle/features/settings/SettingsViewModel.kt]: but again, no sweat on changing imo. unless you want to change it I don't think it hurts code readability or anything

Comment (hskrtich) [reply app/src/main/java/io/github/mattpvaughn/chronicle/features/settings/SettingsViewModel.kt]: Okay fixed lol

**Review by mattttvaughn — APPROVED** [review]

**Pull request merged**